nncf/quantization/algorithms/post_training/algorithm.py

```diff
@@ -202,7 +202,9 @@ def apply(
             for algorithm in self.algorithms:
                 algo_statistic_points = algorithm.get_statistic_points(modified_model, modified_model_graph)
                 statistics_aggregator.register_statistic_points(algo_statistic_points)
-
+            if not statistics_aggregator.statistic_points:
+                nncf_logger.info("The model has no operations to apply quantization.")
+                return modified_model
```

Comment:
Could you please explain why this is required?

Reply:
This is an early exit:
1) to not do extra work.
2) for better logging purposes. Also, in case of having If node, the log will be informative.
![image](https://github.com/openvinotoolkit/nncf/assets/32935044/eea8755f-3fd2-4293-8133-80bfb870f67a)

Reply:
Once again, you are trying to solve the logging problem by adding these checks.

In my opinion, there should be no checks here. If there are no statistics (i.e., it is empty), the MMQ and FBC should not be applied.

Reply:
@andrey-churkin should the user somehow be notified that the model was not quantized?

Reply:
After discussion decided to put logging inside `MinMaxQuantization`